.github/workflows/scorecards-analysis.yml

```diff
@@ -0,0 +1,55 @@
+name: Scorecards supply-chain security
+on:
+  # Only the default branch is supported.
+  branch_protection_rule:
+  schedule:
+    - cron: '22 14 * * 1'
+  push:
+    branches: [ master ]
```

Comment:
Do we want to check master which is the current mobile app or the gekidou branch which is the ipcoming mobile v2?

Reply:
I think you can pick either if you want to -- currently we are adding the scorecards to get a baseline on how things are looking. If we pick one here for the github action, the scorecard thing can be manually and locally run for the other branch as well.

For now master is ok imho as it's out and in production.

Reply:
Yeah I agree for this it probably makes more sense to go with master.